src/main/java/net/sf/jabref/gui/keyboard/KeyBindingWrapper.java

```diff
@@ -0,0 +1,50 @@
+package net.sf.jabref.gui.keyboard;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import org.apache.commons.lang.WordUtils;
+
+public class KeyBindingWrapper {
+
+    private KeyBinding keyBinding;
+    private final SimpleStringProperty bind;
+    private final KeyBindingCategory category;
+
+
+    public KeyBindingWrapper(KeyBinding keyBinding, String bind, KeyBindingCategory category) {
+        this.keyBinding = keyBinding;
+        this.bind = new SimpleStringProperty();
+        setBind(bind);
+        this.category = category;
+    }
+
+    public KeyBinding getKeyBinding() {
+        return keyBinding;
+    }
+
+    public void setKeyBinding(KeyBinding keyBinding) {
+        this.keyBinding = keyBinding;
+    }
+
+    public String getBind() {
+        return bind.get();
+    }
+
+    public StringProperty bindProperty() {
+        return this.bind;
+    }
+
+    public void setBind(String bind) {
+        bind = WordUtils.capitalizeFully(bind);
```

Comment:
No the problem is that we need the keycode combination to be in a blank separated string to ensure compatibility with AWT's `KeyStroke` as well as JavaFX's `KeyCombination`.